docs/titan-basics/workspace-windows.md

```diff
@@ -158,7 +158,7 @@ You can save workspace layouts, allowing you to reconfigure either all the windo
 screen, or all the windows on all the screens, at the touch of a button.
 
 Workspace layouts are saved to the Workspaces touch buttons to the left
-of the A-G menu buttons and above the wheel displays (except on Diamond 9, Sapphire Touch and Titan Go where
+of the A-G menu buttons and above the wheel displays (except on Diamond 9, Diamond 7,  Sapphire Touch and Titan Go where
```

Comment:
for A-G buttons do you mean the softkey buttons? I think these are generally referred to as soft key buttons elsewhere.